Fix travis script

emilevauge opened the pull request:
This PR fixes Travis script issues from #843:
- add `skip_cleanup` in pages & releases
- deploy `traefik*` in releases (in order to deploy `traefik` default linux binary)
- fix provider `script` instead of `scripts`
- fix how yarn is installed
- remove docker 1.13 from tests (really, really, really unstable)
- how do i pronounce this damn project ? 

There are still some strange issue on this new script. For example, this job is green but there is one integration test failing https://travis-ci.org/containous/traefik/jobs/194860155#L2078.

/cc @guilhem @vdemeester 

Signed-off-by: Emile Vauge <emile@vauge.com>

vdemeester commented on .travis.yml:
```diff
@@ -11,60 +11,61 @@ env:
 install:
   - docker version
   - pip install --user -r requirements.txt
+  - 'if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then
```
We do have a travis cron job ? 🤔

emilevauge replied:
We do :)

vdemeester replied:
Nice, I didn't know 😅

Russell-IO commented:
@emilevauge build seems to be stuck here

emilevauge commented:
This PR *should* have been great.
I tried to pre-build base images (including glide and yarn dependencies) using Travis cron in order to get faster builds. But in the end, NOTHING changed. Jobs were still slow as hell.
![](http://i.giphy.com/OMeGDxdAsMPzW.gif)
So I reverted everything and just added few fixes...
